data_steward/common.py

```diff
@@ -30,6 +30,13 @@
 AOU_REQUIRED = [CARE_SITE, CONDITION_OCCURRENCE, DEATH, DEVICE_EXPOSURE, DRUG_EXPOSURE,
                 FACT_RELATIONSHIP, LOCATION, MEASUREMENT, NOTE, OBSERVATION, PERSON,
                 PROCEDURE_OCCURRENCE, PROVIDER, SPECIMEN, VISIT_OCCURRENCE]
+# TODO: Document.
```

Comment:
@ChaoPang I could use help understanding how to document this set of tables accurately - or is there a better name I should choose instead? Or are these really just specific to the person_id_validator tool, and I shouldn't have pulled them out?

For context: my goal was to get a list of all tables which have a person ID in them, where the rows have provenance (indicating EHR vs RDR).

Reply:
@calbach this is a good question. The **person_id_validator** rule checks if the person_id is valid in all the domain tables that contain patient data, which is a subset of the [OMOP Standardized Clinical Data Tables](https://github.com/OHDSI/CommonDataModel/wiki/Standardized-Clinical-Data-Tables). 

1. This is just my personal opinion, I don't think the set of tables in your cleaning rule is the same as **person_id_validator** because it should inform us what AOU required tables contain patient data, whereas **person_id_validator** separated them out into 2 sets. 

2.  In addition, HPO sites will start submitting note data next year so we will have to add **note** and **note_nlp** to this set and MAPPED_VALIDATION_TABLES (or whatever this list should be called) needs to be updated.

Reply:
Thanks, this is very helpful. I've renamed the tables and added documentation. Please let me know if this makes sense to you.